server/src/test/java/io/crate/metadata/sys/SysClusterTableInfoTest.java

```diff
@@ -33,6 +39,31 @@
 
 public class SysClusterTableInfoTest extends CrateDummyClusterServiceUnitTest {
 
+    @Test
+    public void test_routing_allocation_awareness_settings() {
+        var clusterService = new ClusterService(
+            Settings.EMPTY,
+            new ClusterSettings(
+                Settings.builder()
+                    .put(AwarenessAllocationDecider.CLUSTER_ROUTING_ALLOCATION_AWARENESS_ATTRIBUTE_SETTING.getKey(), List.of("zone"))
+                    .put("cluster.routing.allocation.awareness", "force", new String[] {"zone"}, new String[] {"a,b,c"})
+                    .build(),
+                ClusterSettings.BUILT_IN_CLUSTER_SETTINGS),
+            THREAD_POOL);
+        var clusterTable = SysClusterTableInfo.of(clusterService);
+
+        StaticTableReferenceResolver<Void> refResolver = new StaticTableReferenceResolver<>(clusterTable.expressions());
+        NestableCollectExpression<Void, ?> awareness = refResolver.getImplementation(clusterTable.getReference(ColumnIdent.of(
+            "settings",
+                List.of("cluster", "routing", "allocation", "awareness"))));
+        awareness.setNextRow(null);
+        assertThat(awareness.value().toString())
+            .satisfiesAnyOf(
+                s -> assertThat(s).isEqualTo("{attributes=[zone], force={={zone=a,b,c}}}"),
+                s -> assertThat(s).isEqualTo("{force={={zone=a,b,c}}, attributes=[zone]}")
```

Comment:
not the `={={zone` part. this happens because the force is a group setting, and it expects multiple objects under it.